faf.spec.in

```diff
@@ -120,11 +191,7 @@ Requires: python-munch
 Requires: python-dateutil
 Requires: python2-ratelimitingfilter
 Requires: python-simplejson
-Requires: xstatic-patternfly-common
-Requires: nodejs-typeahead.js
-Requires: nodejs-moment
-Requires: js-jquery
-Requires: xstatic-datatables-common
```

Comment:
Why were these packages removed?

Reply:
It is my mistake, I have returned the lines back.